apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -162,16 +162,9 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                 </Tooltip>
             ) : (
                 <Tooltip title="Add custom events">
-                    <Button
-                        disableRipple={true}
-                        onClick={handleOpen}
-                        variant="outlined"
-                        size="small"
-                        startIcon={<Add fontSize="small" />}
-                        disabled={skeletonMode}
-                    >
-                        Custom
-                    </Button>
+                    <IconButton disableRipple={true} onClick={handleOpen} size="medium" disabled={skeletonMode}>
```

Comment:
```suggestion
                    <IconButton onClick={handleOpen} size="medium" disabled={skeletonMode}>
```

issue: I know you didn't write this part, but `disableRipple` messes with the styling on hover and is inconsistent with the other buttons